src/cheri/insns/gcmode_32bit.adoc

```diff
@@ -12,13 +12,22 @@ Mnemonic::
 Encoding::
 include::wavedrom/gcmode.adoc[]
 
-Description::
+*Description*
```

Comment:
```suggestion
Description::
```
This is inconsistent

Reply:
> This is inconsistent

this was due to a formatting nightmare I had - I've given up on numbered lists and typed the numbers manually and the situation has improved now